recipes/rdma-core/all/conanfile.py

```diff
@@ -73,6 +72,10 @@ def generate(self):
         # tc.variables["ENABLE_STATIC"] = not self.options.shared
         tc.variables["NO_PYVERBS"] = True
         tc.variables["NO_MAN_PAGES"] = True
+        # Otherwise get set to ${install_prefix}/car/run and the paths in
+        # https://github.com/linux-rdma/rdma-core/blob/v52.0/buildlib/config.h.in
+        # can exceed the 108-character limit for socket paths on Linux
+        tc.variables["CMAKE_INSTALL_RUNDIR"] = "/var/run"
```

Comment:
```suggestion
        if "CMAKE_INSTALL_RUNDIR" not in self.conf.get("tools.cmake.cmaketoolchain:extra_variables", check_type=dict, default={}):
            tc.variables["CMAKE_INSTALL_RUNDIR"] = self."/var/run"
```

I would keep it more flexible as we have issues when declaring hardcoded system path in a recipe.

Reply:
Thanks! That's a much better solution.